test/cloudtest/pkg/tests/all_cluster_fail_test.go

```diff
@@ -93,15 +93,18 @@ func TestClusterInstancesOnFailGoRunner(t *testing.T) {
 	g.Expect(report.Suites[1].Tests).To(Equal(5))
 	g.Expect(len(report.Suites[1].TestCases)).To(Equal(5))
 
+	foundFailTest := false
+
 	for _, t := range report.Suites[0].TestCases {
 		if t.Name == "_TestFail" {
 			g.Expect(t.Failure).NotTo(Equal(BeNil()))
 			g.Expect(strings.Contains(t.Failure.Contents, ">>>Running on fail script<<<")).To(Equal(true))
-
+			foundFailTest = true
```

Comment:
Is it necessary here? I assume at least cycle can be finished here

Reply:
Yes, it is necessary. It excludes the case of the renaming of the failing test.

Reply:
We have internally discussed. It is should be ok. But let me know if it still is a problem.